CONTRIBUTING.md

```diff
@@ -502,6 +510,45 @@ kubectl -n crossplane-system scale deploy crossplane --replicas=0
 make run
 ```
 
+## Code Review Process
+
+All Pull Requests (PR), whether written by a Crossplane maintainer or a
+community member, must go through code review. This ensures that the code is
+correct, maintainable, and secure.
+
+We encourage anyone in the community to conduct a code review on a PR,
+however, we require the following approvals before merging a PR:
+- At least one approval from [Reviewers] or [Code Owners]
+- At least one approval from [Maintainers]
+
+When opening a PR, GitHub will automatically assign the
+[crossplane-reviewers] and [crossplane-maintainers] teams, as well as
+any [Code Owners]. The PR author may also add any additional reviewers they
+might want to have a look at the PR.
+
+If the PR author has a specific code reviewer in mind, we encourage them to
+remove the auto-assigned reviewers and directly add the individuals they would
+like to review the code.
+
+[Maintainers] will be the last to review a PR, after the initial review from
+[Reviewers] and are the only ones that can merge a PR.
+
+The PR author should notify the [crossplane-maintainers] team or the
+individual maintainer assigned to the PR, when the PR is ready for their final
+review.
```

Comment:
I'd prefer the PR author to follow up with the people who were auto-assigned first, since the assignment is intended to fairly spread review load.

Reply:
Going to remove based on clarified understanding of the assignment rules.